firstvoices/backend/tests/test_apis/test_pages_api.py

```diff
@@ -18,7 +18,7 @@
 
 
 class TestSitePageEndpoint(
-    SiteContentPatchApiTestMixin, BaseControlledSiteContentApiTest
+    SiteContentPatchApiTestMixin, BaseReadOnlyControlledSiteContentApiTest
```

Comment:
Should this use the `BaseControlledSiteContentApiTest` as it is a write API?

Reply:
Yes, sorry about that I misread what base class to use. I made a new base class: `BaseControlledLanguageAdminOnlySiteContentAPITest` that has the correct mixins without the assistant tests, as they will fail for pages and widgets